src/generic_rosenbrock.jl

```diff
@@ -0,0 +1,59 @@
+struct RosenbrockTableau
+    a::Array{Real,2}
+    C::Array{Real,2}
+    b::Array{Real,1}
+    gamma::Real
+    d::Array{Real,1}
+    c::Array{Real,1}
+end
+
+macro _genConstantCache(inds,name,T)
```

Comment:
This is very minor, but I would like to point out that it is recommended to use lowercase letters for functions https://docs.julialang.org/en/v1/manual/variables/#Stylistic-Conventions-1